payments/invoice/seamless-view.md

```diff
@@ -23,107 +23,363 @@ sidebar:
+  request where you send in the consumer data. Use the `FinancingConsumer`
+  activity
+* ⑤ To get the authorization result, you need to follow up with a `GET` request
+  using the paymentID received in the first step.
+
+* Finally, when you are ready to ship your order, you will have to make a `POST`
+  request to make a [Capture][capture]. **At this point Swedbank Pay will
+  generate the invoice to the consumer.**
 
 ## API requests
 
-The API requests are displayed in the [invoice flow](#invoice-flow).
-The options you can choose from when creating a payment with key operation set
-to Value FinancingConsumer are listed below.
+The API requests are displayed in the [invoice flow](#invoice-flow). The options
+you can choose from when creating a payment with key operation set to Value
+`FinancingConsumer` are listed below.
+
+* An invoice payment is always two-phased based - you create an
+  [`Authorize`][authorize] transaction, that is followed by a
+  [`Capture`][capture] or [`Cancel`][cancel] request.
+* **Defining CallbackURL**: When implementing a scenario, it is optional to set
+  a [CallbackURL][callback] in the `POST` request. If callbackURL is set
+  Swedbank Pay will send a postback request to this URL when the consumer has
+  fulfilled the payment. [See the Callback API description here.][callback]
+
+## Seamless View Back End
+
+When properly set up in your merchant/webshop site and the payer starts the
+invoice process, you need to make a POST request towards Swedbank Pay with your
+invoice information. This will generate a payment object with a unique
+`paymentID`. You will receive a **JavaScript source** in response.
 
 ### Options before posting a payment
 
-**POST Request**
+Different countries have different values for the properties. The table below
+showcase the values for the respective countries:
+
+**POST Request Options**
 
 {:.table .table-striped}
-| *POST Request* | **Sweden** ![Swedish flag][se-png] | **Norway** ![Norwegian flag][no-png] | **FInland** ![Finish flag][fi-png] |
+| *Country* | **Sweden** ![Swedish flag][se-png] | **Norway** ![Norwegian flag][no-png] | **FInland** ![Finish flag][fi-png] |
 | *Operation* | FinancingConsumer | FinancingConsumer | FinancingConsumer |
 | *Intent* | Authorization | Authorization | Authorization |
 | *Currency* | SEK | NOK | EUR |
 | *InvoiceType* | PayExFinancingSE | PayExFinancingNO | PayExFinancingFI |
 
-* An invoice payment is always two-phased based - you create an
-    [`Authorize`][authorize] transaction, that is followed by a [`Capture`][capture] or [`Cancel`][cancel] request.
-* **Defining CallbackURL**: When implementing a scenario, it is optional
-    to set a [CallbackURL][callback-api]in the `POST` request.
-    If callbackURL is set PayEx will send a postback request to this URL when
-    the consumer has fulfilled the payment.
-    [See the Callback API description here.][callback-api]
+### Intent
 
-### Invoice flow
+{% include intent.md %}
```

Comment:
The include intent.md only has links for card.

Reply:
Well spotted! The links are relative now